code/modules/surgery/autopsy.dm

```diff
@@ -0,0 +1,35 @@
+/datum/surgery/autopsy
+	name = "Autopsy"
+	steps = list(/datum/surgery_step/incise, /datum/surgery_step/retract_skin, /datum/surgery_step/saw, /datum/surgery_step/clamp_bleeders,
+				 /datum/surgery_step/autopsy, /datum/surgery_step/close)
+	target_mobtypes = list(/mob/living)
+	possible_locs = list(BODY_ZONE_R_ARM,BODY_ZONE_L_ARM,BODY_ZONE_R_LEG,BODY_ZONE_L_LEG,BODY_ZONE_CHEST,BODY_ZONE_HEAD)
+	requires_tech = FALSE
+
+/datum/surgery/autopsy/can_start(mob/user, mob/living/carbon/target)
+	. = ..()
+	if(target.stat != DEAD)
+		return FALSE
+
+/datum/surgery_step/autopsy
+	name = "analyze cause of death"
+	implements = list(TOOL_HEMOSTAT = 90, TOOL_WIRECUTTER = 35, /obj/item/stack/packageWrap = 15, /obj/item/stack/cable_coil = 5)
+	time = 90
+	experience_given = 20
+
+/datum/surgery_step/autopsy/preop(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	user.visible_message("<span class='notice'>[user] starts analyzing [target].</span>", "<span class='notice'>You start analyzing  [target].</span>")
+
+/datum/surgery_step/autopsy/success(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery, default_display_results = FALSE)
+	for(var/obj/O in target.damage_record)
```

Comment:
add a do_after delay or something to these, or else the surgeon's chat could get spammed with messages if someone died a death of a thousand cuts

Reply:
reworked how it shows up -> now it will be in a note. resolved

Reply:
reworked how it shows up -> now it will be in a note. resolved